embed/serve_test.go

```diff
@@ -36,3 +37,33 @@ func TestStartEtcdWrongToken(t *testing.T) {
 		t.Fatalf("expected %v, got %v", auth.ErrInvalidAuthOpts, err)
 	}
 }
+
+// TestStartEtcdLargeBcryptCost ensures that StartEtcd with good configs returns with error.
```

Comment:
trivial nit: `good configs` should be `bad configs`?

Reply:
Updated.